src/transforms/es6/providetomodule.js

```diff
@@ -6,6 +6,22 @@ const {addRequire, isClosureClass, isControllerClass, isDirective, isGoogDefine,
 const {createAssignmentShim, createUIShim} = require('../../utils/shim');
 const {logger} = require('../../utils/logger');
 const {resolveThis} = require('../../utils/resolvethis');
+const {addOSGlobals, isOSGlobal, convertOSGlobal} = require('../../utils/opensphere');
+
+
+// on creation, read ".jscodeshift.json" file(s) from this and local folders
+(function(fs, workspace, filename){
+  fs.readdirSync(workspace).forEach((folder) => {
+    try {
+      const config = JSON.parse(fs.readFileSync(`${workspace}${folder}/${filename}`, 'utf8'));
+      if (config) {
+        addOSGlobals(config['globals']);
+      }
+    } catch (e) {
+      // no file found or couldn't read it
```

Comment:
`fs.existsSync` can be used to check if the file exists before trying to read it. If it exists and the read fails, an error should be thrown so the script exits and the developer knows their config file is invalid.

Reply:
Ah, much better. Ty ty.